test/pytest/q_leaky_relu_test.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from qkeras.qlayers import QActivation
+from qkeras.quantizers import quantized_relu
+from tensorflow.keras.models import Sequential
+
+import hls4ml
+
+test_root_path = Path(__file__).parent
+
+
+def randX(batch_size, N):
+    return np.random.rand(batch_size, N)
+
+
+@pytest.fixture(scope='module')
+def randX_1000_1():
+    return randX(1000, 1)
+
+
+@pytest.mark.parametrize(
+    'quantizer',
+    [
+        (quantized_relu(4, negative_slope=0.5)),
+        (quantized_relu(4, 2, negative_slope=0.5)),
+        (quantized_relu(8, negative_slope=0.125)),
+        (quantized_relu(8, 4, negative_slope=1.0)),
+        (quantized_relu(10, negative_slope=0.25)),
+        (quantized_relu(10, 5, negative_slope=0.5)),
+        (quantized_relu(10, 5, negative_slope=0.25)),
+    ],
+)
+@pytest.mark.parametrize('backend', ['Vivado', 'Vitis', 'Quartus'])
+@pytest.mark.parametrize('io_type', ['io_parallel', 'io_stream'])
+def test_quantizer(randX_1000_1, quantizer, backend, io_type):
```

Comment:
This should have a proper name, and could be fused with the existing qkeras test file. Or split along with the other qactivation tests. But whatever the solution ends up being it should be properly named with `test_` prefix.